src/vue-froala.js

```diff
@@ -89,20 +91,20 @@ export default (Vue, Options = {}) => {
         if (this.editorInitialized) {
           return;
         }
-
         this.currentConfig = this.config || this.defaultConfig;
-
-        this._$element = jQuery(this.$el);
-        
-        
         
+        this._$element = this.$el;
+      
         this.setContent(true);
 
         this.registerEvents();
-        this._$editor = this._$element.froalaEditor(this.currentConfig).data('froala.editor').$el;
+        
+        this._$element.froalaEditor = new FroalaEditor('#'+this._$element.id,this.currentConfig)
```

Comment:
We shall use `this._editor` to store the editor instance. When required, we can access the element by using `this._editor.$el`.